buildkite/bazelci.py

```diff
@@ -213,16 +214,19 @@
         "git_repository": "https://github.com/google/protobuf.git",
         "http_config": "https://raw.githubusercontent.com/bazelbuild/continuous-integration/master/buildkite/pipelines/protobuf-postsubmit.yml",
         "pipeline_slug": "protobuf",
+        "migration_owner": "Bazel Team",
```

Comment:
Do we ever want to have other migration owners?

If no, this field should be a boolean "owned_by_bazel"

If yes, we could use GitHub team names instead.

Reply:
Thanks for the suggestions! I think `owned_by_bazel` is enough for now.